app/templates/main/index.html

```diff
@@ -40,11 +40,41 @@ <h3 id="stock_graph">Alphabet Inc</h3>
           </div>
         </div>
         <div class="card-footer">
-          <strong>
-            This area here will give more details of the stock current values.<br>
-          </strong>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Ab eius ex nihil quam nesciunt architecto corporis fugit, cupiditate hic quidem sapiente, quia mollitia vero dolor harum praesentium quas officiis accusantium! <br>
-          <a href="#">A link to the stock details page</a>
+        <head>
+          <h3>Stock Values</h3>
+          <style>table, th, td{ border: 1px solid black; }</style>
+        </head>
```

Comment:
Why is there a head tag here. This should not be here and can cause errors in browsers.

Reply:
I put it there because "head tag contains the title of the document, styles, scripts, and other HTML elements". In this case Stock values is the subtitle for this section.